### DISCUSSION TITLE
Type accented vowels maintaining english layout

### DISCUSSION THREAD

**Discussion by alejoordonez02** [JaKooLit/Hyprland-Dots#475] [Q&A] [answered]
Hello. I can't find a way to be able to write accented vowels by ralt composing while keeping my english keyboard layout. Thank you in advance.

**Comment by JaKooLit** [comment]
https://github.com/hyprwm/Hyprland/issues/1043

you might want to go through. Sorry cant help much

**Reply by alejoordonez02** [reply, marked as answer]
I managed to solve this by doing super + e -> edit user settings and adding compose:ralt to kb_options under input{}
